pagopa/pn-frontend
# payment detail box - pn-4034

## Short description
This PR introduces the box payment info after the payment is in 'succeeded' status.

## List of changes proposed in this pull request
- Adds NotificationPaidDetail component in pn-commons
- Adds paymentHistory mapping for payment events in parseNotificationDetail in pn-commons
- Adds NotificationPaidDetail in NotificationPayment for recipients apps
- Adds NotificationPaidDetail in NotificationDetail.page.tsx 
- Adds tests for NotificationPaidDetail

## How to test
- In PF with the user fieramosca app search the notification with IUN    QXAE-GYMU-YXPT-202303-T-1 and check that payment info are shown after the success banner
- In PA  with ivan, Comune di Milano, search the notification with IUN    UTPL-UEDT-MVHD-202303-H-1 and check that payment info are shown in the new 'Pagamenti' section
- There are currently no paid notification for PG app, test should be postponed to another instance
